library/ssl_tls13_keys.c

```diff
@@ -1153,6 +1158,10 @@ int mbedtls_ssl_tls1_3_create_psk_binder( mbedtls_ssl_context *ssl,
     mbedtls_md_info_t const *md_info = mbedtls_md_info_from_type( md_type );
     size_t const md_size = mbedtls_md_get_size( md_info );
 
+#if !defined(MBEDTLS_DEBUG_C)
+    ((void) ssl);
```

Comment:
We can further here and set `ssl` to `NULL` to make sure we are not using it, as promised in the documentation.

Reply:
```suggestion
    ssl = NULL;
    ((void) ssl);
```